Header header
float32 minTheta
float32 maxTheta
float32 degreesPerSecond
int32 voxelize
float32 voxelResolution
